src/arrayMethodSort.js

```diff
@@ -4,9 +4,18 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
+  ) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
```

Comment:
The comparison function should return a negative, zero, or positive value. It's important to check for a value greater than 0 explicitly. If the compareFunction returns true or false, it might not work properly for sorting numbers or other types where the comparison is not strictly boolean.